service/core/client/src/main/scala/com/lightbend/lagom/internal/client/CircuitBreakersPanelInternal.scala

```diff
@@ -68,14 +80,19 @@ private[lagom] class CircuitBreakersPanelInternal(
         val callTimeout = breakerConfig.getDuration("call-timeout", MILLISECONDS).millis
         val resetTimeout = breakerConfig.getDuration("reset-timeout", MILLISECONDS).millis
 
+        import scala.collection.JavaConverters.asScalaBufferConverter
+        val exceptionWhitelist: Set[String] = breakerConfig.getStringList("exception-whitelist").asScala.toSet
```

Comment:
I am using a `Set[String]` instead of using a more strongly typed approach because this code sits on `service/core/client`.

I considered an alternative: add a marker interface on `service/core/client` and add that marker interface to all Lagom-provided exceptions that map to 4xx status codes. Then, users would still have to add that marker interface on their code. Finally, users would still have to opt-in to the behaviour where the marker interface is considered whitelist. The current approach, even though it's _stringly-typed_ (aka, not _strongly-typed_) keeps all the setup in a single location and modern IDE's support renaming a `String` on a `.conf` file when the related class is renamed or moved to a different package.